src/common/error_data.cpp

```diff
@@ -44,6 +44,11 @@ ErrorData::ErrorData(const string &message) : initialized(true), type(ExceptionT
 const string &ErrorData::Message() {
 	if (final_message.empty()) {
 		final_message = Exception::ExceptionTypeToString(type) + " Error: " + raw_message;
+		if (type == ExceptionType::INTERNAL) {
+			final_message += "\nThis error signals an assertion failure within DuckDB. This usually occurs due to "
```

Comment:
Maybe we should ask them to provide the data/queries on an issue?

Reply:
Yea we talked about this, and we agree, but we want to be clear about what constitutes a good issue
Knowing something is broken is useful, but that usefulness only goes so far if it doesn't have a reproduction.

We were thinking of elaborating on that in the docs page, rather than sending a wall of text here